src/voxel.h

```diff
@@ -134,7 +134,7 @@ class VoxelArea
 	 */
 	bool hasEmptyExtent() const
 	{
-		return MaxEdge - MinEdge == v3s16(-1, -1, -1);
+		return m_cache_extent == v3s16(0,0,0);
```

Comment:
My IDE search says nobody writes to MinEdge or MaxEdge.

Reply:
Maybe add a comment to `MinEdge`/`MaxEdge` to warn about not writing to them.

(I wish my IDE could do this too.)

Reply:
It might be worth to make `(Max|Min)Edge` private and expose them through functions to ensure there's no-one writing to them. Currently this does appear to be somewhat insecure and error-prone.